README.md

```diff
@@ -116,3 +116,15 @@ You may be able to get away without thinking about all this if your project spec
 Error suggestions may be triggered from some underlined compiler warnings. There
 is no visual indication, currently this will basically be for 'import' warnings,
 and can be triggered by 'alt-enter' (PureScript: Show Quick Fixes).
+
+## Hacking on atom-ide-purescript
+
+Install dependencies
+  • `bower install`
+  • `npm install`
+
+Build: `pulp build`
```

Comment:
I'd remove this. Just need to run one step `npm install` or `npm run bundle`. (or as I prefer `npm run -s bundle`).